examples/multiphysics/3d-dam-break/experimental_data.txt

```diff
@@ -0,0 +1,7396 @@
+Time (s)	H1 (m)	H2 (m)	H3 (m)	H4 (m)
```

Comment:
You could use commas or semicolons to separate the columns and this would make it easier to spot typos, but keeping it this way is just fine.